driver: tiny: remove redundant debug message

nak3 opened the pull request.

nak3 commented:
To check the applied drivers, following log is enough. If `exec` and `qemu` will output the message, other drivers also need it.

~~~
    2016/08/20 18:58:04.554522 [DEBUG] client: applied fingerprints [arch cgroup cpu host memory network nomad storage]
~~~

diptanu commented:
@nak3 We do periodic fingerprinting for exec, so the debug message in the fingerprinter is necessary since the driver can be enabled at a later stage than the initial fingerprinting. However, we could remove the debug messages from Qemu since it's static.

nak3 commented:
@diptanu Thank you. I updated it. And I added one more update. when `qemu` version is not valid, it should not return error, but just output error and return nil.

diptanu commented on client/driver/qemu.go:
```diff
@@ -117,12 +117,12 @@ func (d *QemuDriver) Fingerprint(cfg *config.Config, node *structs.Node) (bool,
 	if len(matches) != 2 {
 		delete(node.Attributes, qemuDriverAttr)
-		return false, fmt.Errorf("Unable to parse Qemu version string: %#v", matches)
+		if !currentlyEnabled {
```
Can we just leave this the way it was? The client which invokes the fingerprinter is going to print the error.

diptanu commented:
@nak3 Sorry for taking a long time to review the PR. Left one more comment.

nak3 commented:
@diptanu Thank you. I think this PR should be closed without merge.
